src/jsd/actuator_offline.h

```diff
@@ -6,29 +6,31 @@
 // Include c then c++ libraries
 
 // Include external then project includes
-#include "fastcat/jsd/actuator.h"
+#include "fastcat/jsd/egd_actuator.h"
 
 namespace fastcat
 {
-class ActuatorOffline : public Actuator
+// ActuatorOffline simulates both EgdActuator and EpdActuator.
+class ActuatorOffline : public EgdActuator
```

Comment:
As soon as a unique state variable is added wot the EPD actuator, this assumption breaks so upon further review, I recommend we add 2 new specializations:

Rename Actuator...
```
class ActuatorBase : public DeviceBase
```
Create ActuatorOfflineBase class for common offline stubs
```
Class ActuatorOfflineBase : public ActuatorBase
// Contains 95% of the shared impl
```
Small Gold-specific specalization
```
class EgdActuatorOffline : public ActuatorOfflineBase
// Populates Gold Drive states appropriately
```
Small Platinum-specific specalization
```
class EpdActuatorOffline : public ActuatorOfflineBase
// Populates Platinum Drive States appropriately
```

Reply:
I inherited from `EgdActuator` because `EpdActuator` uses the profiler in the drive. So `EgdActuator` profiling functions can mimic the `EpdActuator` behavior in offline mode.

I see that `PopulateJsdSlaveConfig` in particular might get wonky if the underlying drive specified in the bus topology is a Platinum because the function accesses `jsd_slave_config_.[egd,epd]`.

One option is to have `EpdActuatorOffline` inherit from `EpdActuator` and override the handling and processing functions of the profile commands to be a copy-paste of the ones in `EgdActuator`.